vars/common-vars.yml

```diff
@@ -1,10 +1,45 @@
 ---
-upstream_repos_clone_folder: /opt
-developer_mode: "{{ (lookup('env','OSH_DEVELOPER_MODE') | default('False', true) ) | bool }}"
-socok8s_workspace: "{{ lookup('env','ANSIBLE_RUNNER_DIR') | default('~/suse-osh-deploy', true) }}"
+upstream_repos_clone_folder: "/opt"
+developer_mode: "{{ (lookup('env','SOCOK8S_DEVELOPER_MODE') | default('False', true) ) | bool }}"
+socok8s_workspace: "{{ lookup('env','ANSIBLE_RUNNER_DIR') | default('~/suse-socok8s-deploy', true) }}"
 socok8s_extravars: "{{ socok8s_workspace }}/env/extravars"
 socok8s_ses_pools_details: "{{ socok8s_workspace }}/ses_config.yml"
 socok8s_registry_cert: "{{ socok8s_workspace }}/certs/domain.crt"
 socok8s_registry_certkey: "{{ socok8s_workspace }}/certs/domain.key"
 socok8s_caasp_environment_details: "{{ socok8s_workspace }}/environment.json"
 socok8s_libvirtuuid: "{{ socok8s_workspace }}/libvirt.uuid"
+
+######################################################
+# Following variables are used by airship logic only
+######################################################
+socok8s_deploy_config_location: "{{ socok8s_workspace }}"
+ucp_namespace_name: "{{ ucp_namespace | default('ucp') }}"
+
+# Variables used for building airship images locally
+suse_distro_identifier: opensuse
+suse_airship_components_base_image: "opensuse/leap:15.0"
```

Comment:
This is kind of absurd. socok8s vars start with socok8s, shouldnt airship vars start with airship? why suse_airship? Do we plan on having airship_* vars coming from a different place?

Reply:
There is no standard naming convention identified and adopted so far. 

May be we can revisit variables used in this file and standardize as a post ucp branch merge.